<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>805</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="qtChartIcons.qrc">
    <normaloff>:/myIcons/chart_icons/line_graph_1.png</normaloff>:/myIcons/chart_icons/line_graph_1.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(147, 147, 147);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="mainInputFrame">
      <property name="minimumSize">
       <size>
        <width>220</width>
        <height>561</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color: #6d6d6d;
color: black;
border-radius: 10px;
}
QLineEdit{
background-color: #6d6d6d;
color: black;
border-radius: 30px;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <spacer name="verticalSpacerAbove">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>127</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QFrame" name="subInputFrame">
         <property name="frameShape">
          <enum>QFrame::Shape::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QFrame" name="inputLabelsFrame">
            <property name="frameShape">
             <enum>QFrame::Shape::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Shadow::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QLabel" name="pythonLabel">
               <property name="font">
                <font>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>PYTHON</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="cppLabel">
               <property name="font">
                <font>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>C++</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="javaLabel">
               <property name="font">
                <font>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>JAVA</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="javaScriptLabel">
               <property name="font">
                <font>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>JAVASCRIPT</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="kotlinLabel">
               <property name="font">
                <font>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>KOTLIN</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="phpLabel">
               <property name="font">
                <font>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>PHP</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="cshLabel">
               <property name="font">
                <font>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>C#</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="htmlLabel">
               <property name="font">
                <font>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>HTML</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="inputLineEditFrame">
            <property name="frameShape">
             <enum>QFrame::Shape::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Shadow::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <widget class="QLineEdit" name="pythonLineEdit">
               <property name="maximumSize">
                <size>
                 <width>115</width>
                 <height>18</height>
                </size>
               </property>
               <property name="text">
                <string>0</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="cppLineEdit">
               <property name="maximumSize">
                <size>
                 <width>115</width>
                 <height>18</height>
                </size>
               </property>
               <property name="text">
                <string>0</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="javaLineEdit">
               <property name="maximumSize">
                <size>
                 <width>115</width>
                 <height>18</height>
                </size>
               </property>
               <property name="text">
                <string>0</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="javaScriptLineEdit">
               <property name="maximumSize">
                <size>
                 <width>115</width>
                 <height>18</height>
                </size>
               </property>
               <property name="text">
                <string>0</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="kotlinLineEdit">
               <property name="maximumSize">
                <size>
                 <width>115</width>
                 <height>18</height>
                </size>
               </property>
               <property name="text">
                <string>0</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="phpLineEdit">
               <property name="maximumSize">
                <size>
                 <width>115</width>
                 <height>18</height>
                </size>
               </property>
               <property name="text">
                <string>0</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="cshLineEdit">
               <property name="maximumSize">
                <size>
                 <width>115</width>
                 <height>18</height>
                </size>
               </property>
               <property name="text">
                <string>0</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="htmlLineEdit">
               <property name="maximumSize">
                <size>
                 <width>115</width>
                 <height>18</height>
                </size>
               </property>
               <property name="text">
                <string>0</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="updatePushButton">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
background-color: #6d6d6d;
color: black;
border-radius: 10px;
}</string>
         </property>
         <property name="text">
          <string>Update Chart</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacerBelow">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>168</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="mainDisplayFrame">
      <property name="font">
       <font>
        <bold>false</bold>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QFrame" name="appInfoFrame">
         <property name="minimumSize">
          <size>
           <width>541</width>
           <height>140</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>140</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QFrame" name="descriptionFrame">
            <property name="frameShape">
             <enum>QFrame::Shape::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Shadow::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QLabel" name="mainDescriptionLabel">
               <property name="font">
                <font>
                 <pointsize>11</pointsize>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>The Most Popular Programming Languages In The World</string>
               </property>
               <property name="indent">
                <number>60</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="subDescriptionLabel">
               <property name="font">
                <font>
                 <pointsize>9</pointsize>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>A Representation of The Most Popular Programming Languages in Pie Chart, Line Graph and Bar Chart Presentations.</string>
               </property>
               <property name="margin">
                <number>0</number>
               </property>
               <property name="indent">
                <number>11</number>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="chartSelectionFrame">
            <property name="styleSheet">
             <string notr="true">QPushButton{
background-color: #6d6d6d;
color: black;
border-radius: 10px;
}</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::Shape::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Shadow::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QPushButton" name="pieChartPushButton">
               <property name="font">
                <font>
                 <pointsize>12</pointsize>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Pie Chart</string>
               </property>
               <property name="icon">
                <iconset resource="qtChartIcons.qrc">
                 <normaloff>:/myIcons/chart_icons/pie_chart_1.png</normaloff>
                 <normalon>:/myIcons/chart_icons/pie_chart_2.png</normalon>:/myIcons/chart_icons/pie_chart_1.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>22</width>
                 <height>25</height>
                </size>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="checked">
                <bool>false</bool>
               </property>
               <property name="autoExclusive">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="lineGraphPushButton">
               <property name="font">
                <font>
                 <pointsize>12</pointsize>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Line Graph</string>
               </property>
               <property name="icon">
                <iconset resource="qtChartIcons.qrc">
                 <normaloff>:/myIcons/chart_icons/line_graph_2.png</normaloff>
                 <normalon>:/myIcons/chart_icons/line_graph_1.png</normalon>:/myIcons/chart_icons/line_graph_2.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>22</width>
                 <height>25</height>
                </size>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="checked">
                <bool>false</bool>
               </property>
               <property name="autoExclusive">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="barChartPushButton">
               <property name="font">
                <font>
                 <pointsize>12</pointsize>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Bar Chart</string>
               </property>
               <property name="icon">
                <iconset resource="qtChartIcons.qrc">
                 <normaloff>:/myIcons/chart_icons/bar_chart_1.png</normaloff>
                 <normalon>:/myIcons/chart_icons/bar_chart_2.png</normalon>:/myIcons/chart_icons/bar_chart_1.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>22</width>
                 <height>25</height>
                </size>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="checked">
                <bool>false</bool>
               </property>
               <property name="autoExclusive">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="chartDisplayFrame">
         <property name="frameShape">
          <enum>QFrame::Shape::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QStackedWidget" name="stackedWidget">
            <property name="currentIndex">
             <number>2</number>
            </property>
            <widget class="QWidget" name="pieChartPage">
             <layout class="QVBoxLayout" name="verticalLayout_8">
              <item>
               <widget class="QLabel" name="pieChartLabel">
                <property name="font">
                 <font>
                  <pointsize>12</pointsize>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="frameShape">
                 <enum>QFrame::Shape::WinPanel</enum>
                </property>
                <property name="text">
                 <string>Pie Chart </string>
                </property>
                <property name="indent">
                 <number>200</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QChartView" name="pieChartGraphicsView">
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(255, 255, 255);</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="lineGraphicsPage">
             <layout class="QVBoxLayout" name="verticalLayout_9">
              <item>
               <widget class="QLabel" name="lineGraphLabel">
                <property name="font">
                 <font>
                  <pointsize>12</pointsize>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="frameShape">
                 <enum>QFrame::Shape::WinPanel</enum>
                </property>
                <property name="text">
                 <string>Line Graph</string>
                </property>
                <property name="indent">
                 <number>200</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QChartView" name="lineGraphGraphicsView">
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(255, 255, 255);</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="barChartPage">
             <layout class="QVBoxLayout" name="verticalLayout_10">
              <item>
               <widget class="QLabel" name="barChartLabel">
                <property name="font">
                 <font>
                  <pointsize>12</pointsize>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="frameShape">
                 <enum>QFrame::Shape::WinPanel</enum>
                </property>
                <property name="text">
                 <string>Bar Chart</string>
                </property>
                <property name="indent">
                 <number>200</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QChartView" name="barChartGraphicsView">
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(255, 255, 255);</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header>qchartview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="qtChartIcons.qrc"/>
 </resources>
 <connections/>
</ui>
